extensions/windows-event-log/ConsumeWindowsEventLog.h

```diff
@@ -97,6 +98,7 @@ class ConsumeWindowsEventLog : public core::Processor
   std::shared_ptr<core::ProcessSessionFactory> sessionFactory_;
   bool renderXML_{};
   MSXML2::IXMLDOMDocumentPtr xmlDoc_;
+  std::atomic<bool> processorStopped_ = false;
```

Comment:
I think either init or onschedule should set it. 

This way restart won't work as expected as it's never set back to false.

Reply:
The name here is odd -- processorStopped implies that the processor was stopped but if it's not started it is not stopped? The nomenclature seems as though it could lead to confusion.